nette/notejam/app/presenters/HomepagePresenter.php

```diff
@@ -0,0 +1,56 @@
+<?php
+
+namespace App\Presenters;
+
+use App\Components\Notes\INotesFactory;
+use App\Helpers\OrderHelper;
+use App\Model\NoteManager;
+use Nette;
+
+
+class HomepagePresenter extends SecuredBasePresenter
+{
+
+	/** @var NoteManager @inject */
+	public $noteManager;
+
+	/** @var INotesFactory @inject */
+	public $notesFactory;
+
+	/** @var object[] */
+	protected $notes;
+
+
+	/********************* Controls ***********************************************************************************/
```

Comment:
Such comments [can be found](https://github.com/nette/forms/blob/master/src/Forms/Form.php#L288) in framework source codes. I used it for better orientation since the Presenters are classes with lots of methods.